addons/units/uniforms/combat_long_plaid.hpp

```diff
@@ -60,6 +60,16 @@ class CLASS(Uniform_Combat_LS_CDBS_GP_TB): CLASS(Uniform_Base) { // CDB = Plaid
         uniformClass = QCLASS(Unit_Combat_LS_CDBS_GP_TB);
     };
 };
+class CLASS(Uniform_Combat_LS_CDBS_BP_BB): CLASS(Uniform_Base) { // CDB = Plaid Dark Blue
+    scope = 2;
+    author = "GilleeDoo";
+    displayName = CSTRING(Uniform_Combat_LS_CDBS_BP_BB);
+    picture = QPATHTOF(UI\uniform_combat_ls_cdbs_bp_ca.paa);
```

Comment:
```suggestion
    picture = QPATHTOF(UI\uniform_combat_ls_cdbs_bp_ca.paa);
```

This file doesn't exist.

Reply:
Asked Mick to make it, will push it when he sends it to me.

Reply:
I should have them, thought I pushed them already, will do tomorrow

Reply:
I forgot to push them...